CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [v0.15.0] - [v0.14.0]
```

Comment:
I think you forgot to add the link for `v0.15.0` at the end

Reply:
Oh shoot.. fixed it.